modules/odc-adding-a-GitHub-repository-containing-pipelines.adoc

```diff
@@ -51,3 +51,8 @@ image::Git-access-token-secret.png[]
 . Navigate back to the developer console and click *Add*.
 . Read the details of the steps that you have to perform and click *Close*.
 . View the details of the repository you just created.
+
+[NOTE]
+====
+When importing an application using *Import from Git* and the Git repository has a '.tekton` directory, you can configure `pipelines-as-code` for your application. For more information see, xref:../../applications/creating_applications/odc-creating-applications-using-developer-perspective.adoc#odc-creating-applications-using-the-developer-perspective[Creating applications using the Developer perspective].
```

Comment:
This formatting looks wrong (can you add a link to the preview?) - 

```
has a '.tekton` directory,
```

Reply:
@gabriel-rh You are right, the correct formatting should be `.tekton` instead '.tekton`. I have made this change

Sharing the doc preview link as requested: https://56454--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/working-with-pipelines-using-the-developer-perspective.html#odc-adding-a-GitHub-repository-containing-pipelines_working-with-pipelines-using-the-developer-perspective